hibikase/PlaybackWidget.h

```diff
@@ -63,22 +67,25 @@ class PlaybackWidget : public QWidget
     void LoadAudio(std::unique_ptr<QIODevice> io_device);  // Can be nullptr
 
 signals:
-    void TimeUpdated(std::chrono::milliseconds ms);
+    void TimeUpdated(std::chrono::microseconds current);
```

Comment:
What do you think about keeping this one as `milliseconds`? That way we only have to do one cast (inside this class) instead of adding casts in several other places in the program.